code/modules/reagents/reagent_containers.dm

```diff
@@ -16,6 +16,8 @@
 	var/spillable = FALSE
 	var/list/fill_icon_thresholds = null
 	var/fill_icon_state = null // Optional custom name for reagent fill icon_state prefix
+	///The temperature the reagents in list_reagents should be when created
+	var/starting_temperature = DEFAULT_REAGENT_TEMPERATURE
```

Comment:
It'd be nice to use this as a default, and then if the object was crafted or some such, use the temperature of the materials. Realize that's more work tho